kind/values.yaml

```diff
@@ -1,6 +1,6 @@
 orb:
   defaults:
-    logLevel: "info"
+    logLevel: "debug"
```

Comment:
was that intentional?

Reply:
it is, kind for local development, can be debug, we will store only local logs and it is purged after the deletion of the cluster